robot_sf/benchmark/live_forecast_replay_gate.py

```
@@ -0,0 +1,895 @@
+
+        try:
+            forecast_report = evaluate_forecast_batch(
+                batch,
+                {
+                    actor_id: np.asarray(positions, dtype=float)
+                    for actor_id, positions in ground_truth.items()
+                },
+            )
+            forecast_status = "ok"
+            forecast_error = None
+        except Exception as exc:  # pragma: no cover - defensive
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)